media-tv/tvbrowser-bin/tvbrowser-bin-4.2.7.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit desktop
+
+DESCRIPTION="Themeable and easy to use TV Guide - written in Java"
+HOMEPAGE="https://www.tvbrowser.org"
+MY_PN="tvbrowser"
+SRC_URI="mirror://sourceforge/project/tvbrowser/TV-Browser%20Releases%20%28Java%2011%20and%20higher%29/${PV}/${MY_PN}_${PV}_bin.tar.gz -> ${P}.tar.gz"
```

Comment:
Let's try to inline `mirror://sourceforge`

Reply:
I do not understand, what do you mean?

Reply:
Replacing `mirror://sourceforge` with a link of the form `https://sourceforge.net/projects/tvbrowser/files/....tar.gz/download`

`mirror://sourceforge` is no longer useful and is slowly being phased out.  Sometimes pkgcheck warns about it causing a permanent redirect.

Reply:
I have changed it to a https link and tested whether it works.